### DISCUSSION TITLE
Notification filter ‘releases’

### DISCUSSION THREAD

**Discussion by cafferata** [community/community#53904] [Mobile]
I would like to have an (extra) filter inside the GitHub mobile notifications tab for the 'release' type (_similar to desktop_).

1. https://github.com/notifications?query=is%3Arelease
1. https://github.com/notifications?query=is%3Arelease+is%3Aunread

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.110.0 (97380870)
Device: iPhone 13 (iOS 16.3.1)
</details>

**Comment by taki-on** [comment]
Thank you for your suggestion! I will add this to our feature request list and investigate what we can do.
